modules/node_groups/locals.tf

```diff
@@ -34,6 +34,9 @@ locals {
       metadata_http_tokens                 = var.workers_group_defaults["metadata_http_tokens"]
       metadata_http_put_response_hop_limit = var.workers_group_defaults["metadata_http_put_response_hop_limit"]
       ami_is_eks_optimized                 = true
+      user_data_mime_type                  = "text/x-shellscript"
+      user_data_template_extra_args        = lookup(var.workers_group_defaults, "userdata_template_file", {})
```

Comment:
I think the lookup reference to `userdata_template_file` here was meant to be `user_data_template_extra_args`

Reply:
You're right, this is incorrect, but [it should actually be `user data_template_extra_args`](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/locals.tf#L255-L259).